diff --git a/node_modules/govuk-frontend/govuk/.DS_Store b/node_modules/govuk-frontend/govuk/.DS_Store
new file mode 100644
index 0000000..415d465
Binary files /dev/null and b/node_modules/govuk-frontend/govuk/.DS_Store differ
diff --git a/node_modules/govuk-frontend/govuk/components/.DS_Store b/node_modules/govuk-frontend/govuk/components/.DS_Store
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/govuk-frontend/govuk/components/custom/.DS_Store b/node_modules/govuk-frontend/govuk/components/custom/.DS_Store
new file mode 100644
index 0000000..1ec2ccc
Binary files /dev/null and b/node_modules/govuk-frontend/govuk/components/custom/.DS_Store differ
diff --git a/node_modules/govuk-frontend/govuk/components/custom/actionLinks/macro.njk b/node_modules/govuk-frontend/govuk/components/custom/actionLinks/macro.njk
new file mode 100644
index 0000000..06283d9
--- /dev/null
+++ b/node_modules/govuk-frontend/govuk/components/custom/actionLinks/macro.njk
@@ -0,0 +1,3 @@
+{% macro actionLinks(params) %}
+  {%- include "./template.njk" -%}
+{% endmacro %}
diff --git a/node_modules/govuk-frontend/govuk/components/custom/actionLinks/template.njk b/node_modules/govuk-frontend/govuk/components/custom/actionLinks/template.njk
new file mode 100644
index 0000000..2e7f550
--- /dev/null
+++ b/node_modules/govuk-frontend/govuk/components/custom/actionLinks/template.njk
@@ -0,0 +1,39 @@
+{% set actions = params.actions %}
+{% set links = actions.links %}
+{% set title = actions.title %}
+{% if links | length == 1 %}
+  {% set link = links | first %}
+  {% if link.isButton %}
+    <a href="{{ link.href }}" class="govuk-button" data-module="govuk-button">{{ link.text }}</a>
+  {% else %}
+    <div style="margin-bottom: 5px;">
+      {% if link.html %}
+        {{ link.html | safe if link.html else heading.main }}
+      {% else %}
+        <a class="small-link" href="{{ link.href }}">{{ link.text }}</a><br/>
+      {% endif %}
+    </div>
+  {% endif %}
+{% else %}
+  {% for link in links %}
+    {% if link %}
+      {% set isButton = link.isButton %}
+      {% set linkHref = link.href %}
+      {% set linkText = link.text %}
+      {% set linkHtml = link.html %}
+      {% if link.isButton %}
+        <a href="{{ link.href }}" class="govuk-button" data-module="govuk-button">{{ link.text }}</a>
+        <hr/>
+        <h2 class="govuk-heading-l">{{ title }}</h3>
+      {% else %}
+        <div style="margin-bottom: 5px;">
+          {% if link.html %}
+            {{ link.html | safe if link.html else heading.main }}
+          {% else %}
+            <a class="small-link" href="{{ link.href }}">{{ link.text }}</a><br/>
+          {% endif %}
+        </div>
+      {% endif %}
+    {% endif %}
+  {% endfor %}
+{% endif %}
\ No newline at end of file
diff --git a/node_modules/govuk-frontend/govuk/components/custom/badgePanel/macro.njk b/node_modules/govuk-frontend/govuk/components/custom/badgePanel/macro.njk
new file mode 100644
index 0000000..9a7358a
--- /dev/null
+++ b/node_modules/govuk-frontend/govuk/components/custom/badgePanel/macro.njk
@@ -0,0 +1,3 @@
+{% macro badgePanel(params) %}
+  {%- include "./template.njk" -%}
+{% endmacro %}
diff --git a/node_modules/govuk-frontend/govuk/components/custom/badgePanel/template.njk b/node_modules/govuk-frontend/govuk/components/custom/badgePanel/template.njk
new file mode 100644
index 0000000..89eaa0b
--- /dev/null
+++ b/node_modules/govuk-frontend/govuk/components/custom/badgePanel/template.njk
@@ -0,0 +1,25 @@
+{% set title = params.title %}
+{% set text = params.text %}
+{% set locked = params.locked %}
+
+<div class="govuk-panel govuk-panel--confirmation" style="padding:15px; {% if locked %} background: #EAEAEA; {% else %} background: #28A197 {% endif %} ">
+  <div style="display: flex;">
+    <div style="padding-right: 15px;">
+      {% if locked %}
+        <svg style="padding-top:5px;" width="66" height="82" viewbox="0 0 66 82" fill="none" xmlns="http://www.w3.org/2000/svg">
+          <path d="M6.50435 15.4135L9.08696 26.3216L6.50435 37.1822L16.1174 43.063L22 52.5957L33 50.0347L44 52.5957L49.8826 43.063L59.4957 37.1822L56.913 26.3216L59.4957 15.4135L49.9304 9.53268L44 0L33 2.60844L22.0478 0.0474262L16.0696 9.53268L6.50435 15.4135ZM33.0478 45.4818C27.9741 45.4818 23.1082 43.4831 19.5206 39.9254C15.9329 36.3678 13.9174 31.5426 13.9174 26.5113C13.9174 21.48 15.9329 16.6548 19.5206 13.0971C23.1082 9.53945 27.9741 7.54077 33.0478 7.54077C43.5696 7.54077 52.1304 16.0301 52.1304 26.4639C52.1304 36.9925 43.5696 45.4818 33.0478 45.4818ZM32.9522 40.7392C25.013 40.7392 18.6522 34.384 18.6522 26.5113C18.6522 18.6859 25.013 12.2834 32.9522 12.2834C40.8913 12.2834 47.3478 18.6859 47.3478 26.5113C47.3478 34.384 40.8913 40.7392 32.9522 40.7392ZM51.3174 45.956L45.1957 56.5795L35.2478 54.3505L47.3478 82L54.0435 71.5662H66L51.3174 45.956ZM14.4913 46.288L20.4696 56.9589L30.6565 54.5402L18.6522 82L11.9565 71.5662H0L14.4913 46.288Z" fill="#0B0C0C"/>
+        </svg>
+      {% else %}
+        <svg width="66" height="82" viewbox="0 0 66 82" fill="none" xmlns="http://www.w3.org/2000/svg">
+          <path d="M6.50435 15.4135L9.08696 26.3216L6.50435 37.1822L16.1174 43.063L22 52.5957L33 50.0347L44 52.5957L49.8826 43.063L59.4957 37.1822L56.913 26.3216L59.4957 15.4135L49.9304 9.53268L44 0L33 2.60844L22.0478 0.0474262L16.0696 9.53268L6.50435 15.4135ZM33.0478 45.4818C27.9741 45.4818 23.1082 43.4831 19.5206 39.9254C15.9329 36.3678 13.9174 31.5426 13.9174 26.5113C13.9174 21.48 15.9329 16.6548 19.5206 13.0971C23.1082 9.53945 27.9741 7.54077 33.0478 7.54077C43.5696 7.54077 52.1304 16.0301 52.1304 26.4639C52.1304 36.9925 43.5696 45.4818 33.0478 45.4818ZM32.9522 40.7392C25.013 40.7392 18.6522 34.384 18.6522 26.5113C18.6522 18.6859 25.013 12.2834 32.9522 12.2834C40.8913 12.2834 47.3478 18.6859 47.3478 26.5113C47.3478 34.384 40.8913 40.7392 32.9522 40.7392ZM51.3174 45.956L45.1957 56.5795L35.2478 54.3505L47.3478 82L54.0435 71.5662H66L51.3174 45.956ZM14.4913 46.288L20.4696 56.9589L30.6565 54.5402L18.6522 82L11.9565 71.5662H0L14.4913 46.288Z" fill="white"/>
+        </svg>
+      {% endif %}
+    </div>
+    <div class="govuk-panel__body" style="text-align: left; font-size: 1.1rem; {% if locked %} color: #0B0C0C; {% endif %} ">
+      <span style="font-weight: bold; font-size: 1.2rem;">{{ title }}</span>
+      <br/>
+      <br/>
+      {{ text }}
+    </div>
+  </div>
+</div>
\ No newline at end of file
diff --git a/node_modules/govuk-frontend/govuk/components/custom/headingSuper/macro.njk b/node_modules/govuk-frontend/govuk/components/custom/headingSuper/macro.njk
new file mode 100644
index 0000000..7f08c43
--- /dev/null
+++ b/node_modules/govuk-frontend/govuk/components/custom/headingSuper/macro.njk
@@ -0,0 +1,3 @@
+{% macro headingSuper(params) %}
+  {%- include "./template.njk" -%}
+{% endmacro %}
diff --git a/node_modules/govuk-frontend/govuk/components/custom/headingSuper/template.njk b/node_modules/govuk-frontend/govuk/components/custom/headingSuper/template.njk
new file mode 100644
index 0000000..513fedb
--- /dev/null
+++ b/node_modules/govuk-frontend/govuk/components/custom/headingSuper/template.njk
@@ -0,0 +1,9 @@
+{% set text = params.text %}
+
+<h2 
+  style="color: #646A6E; 
+  font-weight: 300; 
+  margin-bottom: 5px;" 
+  class="govuk-heading-l">
+  {{ text }}
+</h2>
\ No newline at end of file
diff --git a/node_modules/govuk-frontend/govuk/components/custom/helpLinks/macro.njk b/node_modules/govuk-frontend/govuk/components/custom/helpLinks/macro.njk
new file mode 100644
index 0000000..b830c97
--- /dev/null
+++ b/node_modules/govuk-frontend/govuk/components/custom/helpLinks/macro.njk
@@ -0,0 +1,3 @@
+{% macro helpLinks(params) %}
+  {%- include "./template.njk" -%}
+{% endmacro %}
diff --git a/node_modules/govuk-frontend/govuk/components/custom/helpLinks/template.njk b/node_modules/govuk-frontend/govuk/components/custom/helpLinks/template.njk
new file mode 100644
index 0000000..81d45c5
--- /dev/null
+++ b/node_modules/govuk-frontend/govuk/components/custom/helpLinks/template.njk
@@ -0,0 +1,26 @@
+{% for help in params.helpSections %}
+  {% if help %}
+    {% set title = help.title %}
+    {% set description = help.description %}
+    {% set links = help.links %}
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+          <hr style="border: 5px solid #005DA6; margin-bottom: 15px" />
+          <h2 class="govuk-heading-l">{{ title }}</h2>
+          <p>{{ description }}</p>
+          {% for link in links %}
+            {% if link %}
+              <div {% if link.id %} id="{{ link.id }}" {% endif %} style="margin-bottom: 5px;">
+                {% if link.html %}
+                  {{ link.html | safe if link.html else heading.main }}
+                {% else %}
+                  <a class="small-link" {% if link.href %} href="{{ link.href }}" {% endif %} {% if link.onClick %} onClick="{{ link.onClick }}" {% endif %}  >{{ link.text }}</a><br />
+                {% endif %}
+              </div>
+            {% endif %}
+          {% endfor %}
+        </div>
+      </div>
+      <br />
+  {% endif %}
+{% endfor %}
\ No newline at end of file
diff --git a/node_modules/govuk-frontend/govuk/components/custom/sideLinks/macro.njk b/node_modules/govuk-frontend/govuk/components/custom/sideLinks/macro.njk
new file mode 100644
index 0000000..137b83d
--- /dev/null
+++ b/node_modules/govuk-frontend/govuk/components/custom/sideLinks/macro.njk
@@ -0,0 +1,3 @@
+{% macro sideLinks(params) %}
+  {%- include "./template.njk" -%}
+{% endmacro %}
\ No newline at end of file
diff --git a/node_modules/govuk-frontend/govuk/components/custom/sideLinks/template.njk b/node_modules/govuk-frontend/govuk/components/custom/sideLinks/template.njk
new file mode 100644
index 0000000..285ef88
--- /dev/null
+++ b/node_modules/govuk-frontend/govuk/components/custom/sideLinks/template.njk
@@ -0,0 +1,26 @@
+{% for help in params.helpSections %}
+  {% if help %}
+    {% set title = help.title %}
+    {% set description = help.description %}
+    {% set links = help.links %}
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
+          <hr style="border: 5px solid #005DA6; margin-bottom: 15px" />
+          <h2 class="govuk-heading-l">{{ title }}</h2>
+          <p>{{ description }}</p>
+          {% for link in links %}
+            {% if link %}
+              <div {% if link.id %} id="{{ link.id }}" {% endif %} style="margin-bottom: 5px;">
+                {% if link.html %}
+                  {{ link.html | safe if link.html else heading.main }}
+                {% else %}
+                  <a class="small-link" {% if link.href %} href="{{ link.href }}" {% endif %} {% if link.onClick %} onClick="{{ link.onClick }}" {% endif %}  >{{ link.text }}</a><br />
+                {% endif %}
+              </div>
+            {% endif %}
+          {% endfor %}
+        </div>
+      </div>
+      <br />
+  {% endif %}
+{% endfor %}
\ No newline at end of file
